ls1intum/ArTEMiS
# Feature/use translations in all pages

### Checklist

### Motivation and Context
Translate all missing pages into German so that we can offer users to switch the language

### Description
add German translations and make them accessible in most HTML files

### Steps for Testing
1. Log in to ArTEMiS
2. Change the Language to German in the Profile
3. Check any page for missing or wrong translations
